### PR TITLE
Sprint 1/issue 2#connection with database

### LINKED ISSUE #2: Connection with database
## REQUIREMENTS / DESCRIPTION

>Create database on firebase and create basic connection with a basic component
### Tasks
- [x] Understand basic concepts about firebase
- [x] Investigate how to authenticate users with firebase
- [x] Configure new database 
- [x] Create connection with the project

### PR DESCRIPTION
**Pull request by Alfredo20164920**
# Description

This PR have the basic configuration with firebase, and new features to improve our code
- Create string connection with database,
- Add .env file to save the secret secrets variables, also, create .env.example file, this file it was upload to show the configuration used in the project
- Installing new dependencies (firebase and dotenv)
- Modified home page with a single test text
- Eslint file modified to delete "CR" error

Co-Authored-By: AlbertoG22 <[albertogarciamartinez99@gmail.com](mailto:albertogarciamartinez99@gmail.com)>
Co-authored-by: Montserrat Aguilar <[aguilarvalle.montse@gmail.com](mailto:aguilarvalle.montse@gmail.com)>

### CODE REVIEW

**Review by Eduardoespl — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 module.exports = {
   root: true,
   extends: '@react-native',
+  rules: {
+    'prettier/prettier': ['error', {endOfLine: 'auto'}],
+  },
```
Comment (ecruz17) [inline .eslintrc.js:6]: nice!

File: src/db/firebase.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import {initializeApp} from 'firebase/app';
+import {getFirestore} from 'firebase/firestore';
+import 'dotenv/config';
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
+  apiKey: process.env.API_KEY,
```
Comment (ecruz17) [inline src/db/firebase.ts:11]: great use of env variables

**Review by ecruz17 — APPROVED** [review]

**Review by DerekAyala — APPROVED** [review]
LGTM!

**Review by Eduardoespl — APPROVED** [review]
LGTM!

**Review by wiltrouble — APPROVED** [review]

**Pull request merged**